src/lib/constants/resourceTypes.ts

```diff
@@ -31,6 +31,9 @@ export const PARAGRAPH_RESOURCE_TYPES = {
   // STAFF_PROFILE: 'paragraph--staff_profile',
   TABLE: 'paragraph--table',
   WYSIWYG: 'paragraph--wysiwyg',
+  QA_PARAGRAPH: 'paragraph--q_a',
```

Comment:
I think I would prefer `QA` rather than `QA_PARAGRAPH`. 

We discussed naming a bit previously, so I apologize if this conflicts with anything I previously shared. But everything in here is a paragraph so it feels confusing to see `QA_PARAGRAPH` and then `QA_SECTION` and `QA_GROUP`. I actually wondered for a second if all of these were types of paragraphs.